java-8-sample/src/main/java/com/microsoft/sample/java8ASCdemo/Java8AscDemoApplication.java

```diff
@@ -0,0 +1,31 @@
+package com.microsoft.sample.java8ASCdemo;
```

Comment:
This repo is under MIT license. Could you add license header like [here](https://github.com/Azure-Samples/Azure-Spring-Cloud-Samples/blob/master/java-11-sample/src/main/java/com/microsoft/sample/Application.java#L1-L5)?